src/test/java/seedu/guest/storage/JsonSerializableGuestBookTest.java

```diff
@@ -25,8 +25,8 @@ public void toModelType_typicalPersonsFile_success() throws Exception {
         JsonSerializableGuestBook dataFromFile = JsonUtil.readJsonFile(TYPICAL_PERSONS_FILE,
```

Comment:
Should this be TYPICAL_GUESTS_FILE, along with changing lines 19-21 from PERSONS_FILE to GUESTS_FILE?